tests/client_test.py

```diff
@@ -43,53 +35,55 @@ def test_unrequired_param_not_added_to_request_when_none(self):
             mock_add_param.assert_called_once_with(param, False, mock_request)
 
 
-class SwaggerClientFactoryTest(unittest.TestCase):
+class SwaggerClientCacheTest(unittest.TestCase):
     """Test the proxy wrapper of SwaggerClient
     """
 
     def setUp(self):
-        client.factory = None
+        client.cache = None
 
-    def test_is_stale_returns_true_after_timeout(self):
+    tearDown = setUp
```

Comment:
These tests mutate the global cache. They were cleaning up before running, but any other test that happens to hit the cache could fail if the test ordering changes.

This way the global state is reset after the tests as well.

Technically this could only be in tearDown and not setUp, but I'm not sure if all other tests honor this, but I left it in both for now